R/toHTML.R

```diff
@@ -94,6 +94,12 @@ toHTML.default <- function(x, ...) {
   shiny::tags$pre(flextable::htmltools_value(to_flextable(x)))
 }
 
+#' @method .toHTML condition
+#' @keywords internal
+.toHTML.condition <- function(x, ...) {
+  conditionMessage(x)
```

Comment:
Maybe we should wrap this around a `pre` HTML element?

```suggestion
  shiny::tags$pre(conditionMessage(x))
```